dashboards/google-cloud-composer/cloud-composer-monitoring.json

```diff
@@ -0,0 +1,412 @@
+{
+  "category": "CUSTOM",
+  "dashboardFilters": [
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "project_id",
+      "templateVariable": ""
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "environment_name",
+      "templateVariable": ""
+    }
+  ],
+  "displayName": "cloud-composer-monitoring",
```

Comment:
This should be Cloud Composer Monitoring to match line 5 of metadata.yaml

Reply:
Completed